docsrc/installation.rst

```diff
@@ -1,47 +1,83 @@
+CmdStanPy is a pure-Python3 package which wraps CmdStan,
+the command-line interface to Stan which is written in C++,
+therefore, in addition to Python3,
+CmdStanPy requires a modern C++ toolchain in order to build and run Stan models.
+There are several ways to install CmdStanPy and the underlying CmdStan components.
 
+* You can download CmdStanPy, CmdStan, and the C++ toolchain from conda-forge.
 
-Conda users (Recommended)
--------------------------
+* You can download the CmdStanPy package from `PyPI <https://pypi.org>`_
+  using `pip <https://pip.pypa.io/en/stable/>`_.
+  
+* If you want the current development version, you can clone the
+  GitHub `CmdStanPy <https://github.com/stan-dev/cmdstanpy>`_ repository.
 
-If you use `conda <https://docs.conda.io/en/latest/>`__,
-installation of both can be done very simply. CmdStanPy
-and CmdStan are both available via the
-`conda-forge <https://conda-forge.org/>`__ repository.
+If you install CmdStanPy from PyPI or GitHub you will need to
+install CmdStan as well, see section :ref:`CmdStan Installation <cmdstan-install>` below.
+
+  
+Conda: install CmdStanPy, CmdStan, C++ toolchain
+------------------------------------------------
+
+If you use `conda <https://docs.conda.io/en/latest/>`_,
+you can install CmdStanPy and the underlying CmdStan components from the
+`conda-forge <https://conda-forge.org/>`_ repository
+via the following command:
 
-We recommend creating a new environment for CmdStan[Py]:
 
 .. code-block:: bash
 
-    conda create -n cmdstan -c conda-forge cmdstanpy
+    conda create -n stan -c conda-forge cmdstanpy
+
+
+This command creates a new conda environment named ``stan`` and 
+downloads and installs the ``cmdstanpy`` package from conda-forge.
+
+To install into an existing environment, use the conda ``install`` command instead of ``create``:
 
-but installation is possible in an existing environment
-(Note: you must re-activate the environment after running this command!):
 
 .. code-block:: bash
 
     conda install -c conda-forge cmdstanpy
 
-These commands will install CmdStanPy, CmdStan, and the
-required compilers for using CmdStan on your system inside
-a conda environment. To use them, run ``conda activate cmdstan``,
-or whichever name you used for your environment (following ``-n``
-above).
-
-Note that CmdStan is only available on conda for versions
-2.26.1 and newer. If you require an older version, you must use
-one of the following methods to install it. If you require a
-version of CmdStan *newer* than 2.26.1", but not the latest,
-you can install it in the standard conda way by specifying
+Whichever installation method you use, afterwards you must
+activate the new environment or deactivate/activate the existing one.
+For example, if you installed cmdstanpy into a new environment ``stan``,
+run the command
+
+.. code-block:: bash
+
+    conda activate stan
+
+
+The conda package manager will install CmdStan and the C++ toolchain.
```

Comment:
I would move this line up to above the activate command so it is paired with the note on installing cmdstanpy